syncstorage/storage/sql/dbconnect.py

```diff
@@ -363,6 +379,7 @@ def __init__(self, sqluri, create_tables=False, pool_size=100,
             collections.create(self.engine, checkfirst=True)
             user_collections.create(self.engine, checkfirst=True)
             batch_uploads.create(self.engine, checkfirst=True)
+            migration.create(self.engine, checkfirst=True)
```

Comment:
@Micheletto How have we done migrations (if any) in the past? Is `create_tables = true` (triggering this block) always specified in the config file?

Reply:
From what I understand, migrations aren't always triggered, which is why i trap around any call to the migration table. The table can also be built by the migration script, which we will have a bit more control over.